lib/src/screens/dashboard/widgets/sync_indicator_icon.dart

```diff
@@ -46,10 +48,14 @@ class SyncIndicatorIcon extends StatelessWidget {
     return Container(
       height: size,
       width: size,
-      decoration: BoxDecoration(
+      decoration: showTorIcon ? null : BoxDecoration(
         shape: BoxShape.circle,
         color: indicatorColor,
       ),
+      child: !showTorIcon ? null : SvgPicture.asset(
+        "assets/images/tor.svg",
+        color: indicatorColor,
+      ), 
```

Comment:
just better readability 😅 
```suggestion
      child: showTorIcon ? SvgPicture.asset(
        "assets/images/tor.svg",
        color: indicatorColor,
      ) : null, 
```